apps/designer/app/designer/features/workspace/canvas-tools/text-toolbar/text-toolbar.tsx

```diff
@@ -49,16 +64,57 @@ const onClickPreventDefault: MouseEventHandler<HTMLDivElement> = (event) => {
 
 type ToolbarProps = {
   css?: CSS;
-  onValueChange: (value: Value) => void;
   rootRef: React.Ref<HTMLDivElement>;
+  textToolbar: TextToolbarValue;
+  publish: Publish;
 };
 
-const Toolbar = ({ css, onValueChange, rootRef }: ToolbarProps) => {
+const Toolbar = ({ css, rootRef, textToolbar, publish }: ToolbarProps) => {
+  const value: Value[] = [];
+  if (textToolbar.isBold) {
+    value.push("Bold");
+  }
+  if (textToolbar.isItalic) {
+    value.push("Italic");
+  }
+  if (textToolbar.isLink) {
+    value.push("Link");
+  }
   return (
     <ToggleGroup.Root
       ref={rootRef}
-      type="single"
-      onValueChange={onValueChange}
+      type="multiple"
+      value={value}
+      onValueChange={(newValues: Value[]) => {
+        // TODO refactor with per button callback
```

Comment:
nit: we use `@todo`

Reply:
I use TODO because it's hightlighted in vim out of the box
Is `@todo` a vscode thing?

Reply:
I believe it originates in jsdoc